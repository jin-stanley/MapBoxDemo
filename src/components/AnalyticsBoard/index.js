import React from "react";

const AnalyticsBoard = () => {
  return "AnalyticsBoard";
};

export default AnalyticsBoard;
