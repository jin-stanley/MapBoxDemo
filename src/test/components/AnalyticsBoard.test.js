import React from "react";
import renderer from "react-test-renderer";

import AnalyticsBoard from "../../components/AnalyticsBoard";

it("AnalyticsBoard can render success", () => {
  const tree = renderer.create(<AnalyticsBoard />).toJSON();
  expect(tree).toMatchSnapshot();
});
