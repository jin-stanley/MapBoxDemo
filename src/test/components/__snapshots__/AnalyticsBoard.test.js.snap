// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnalyticsBoard can render success 1`] = `"AnalyticsBoard"`;
